lib/index.js

```diff
@@ -409,6 +409,12 @@ Mongoose.prototype.connect = function(uri, options, callback) {
   const _mongoose = this instanceof Mongoose ? this : mongoose;
   const conn = _mongoose.connection;
 
+  if (_mongoose.options.strictQuery === undefined) {
+    utils.warn('Mongoose: the `strictQuery` option will be switched back to `false` by default ' +
+              'in Mongoose 7. Use `mongoose.set(\'strictQuery\', false);` if you want to prepare ' +
+              'for this change.');
```

Comment:
Add "Or use `mongoose.set('strictQuery', true)` to suppress this warning." so we can allow people to suppress this warning if necessary.

Reply:
working fine use mongoose.set() method

Reply:
yes, it works use mongoose.set() method

Reply:
many thanks, its work